src/modules/ROOT/pages/changelog/202209/09.json

```diff
@@ -25,11 +25,26 @@
         "to allow users to map values to record and user fields when importing records."
       ],
       "how": [
-        "by enabling the code that was preventing the users of mapping values to record and user fields."
+        "by removing the code that was preventing the users from mapping values to record and user fields."
       ],
       "refs": [
         "https://github.com/cortezaproject/corteza/commit/0cf81dc45832831da675b2484d0a41c853807802"
       ]
+    },
+    {
+      "what": [
+        "`Cannot read properties of undefined` error when trying to fetch workflow triggers without providing constraints."
+      ],
+      "why": [
+        "to prevent users from getting a console error when fetching workflow triggers without provided constraints."
+      ],
+      "how": [
+        "by adding a check to determine if a constant has a name.",
+        "Also, the automatic addition of constraints after an event type was selected is removed."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/7a9802a1c74da0e58c6bf9e3b24767d80ab970e9"
+      ]
     }
```

Comment:
Isn't this a fix?
In my comment `is this part a change rather than a fix?` I was only referring to the second part (`"Also, the automatic addition of constraints after an event type was selected is removed."`), not the entire thing